Sources/Sentry/SentryFileManager.m

```diff
@@ -149,14 +149,10 @@ - (NSString *)storeData:(NSData *)data toPath:(NSString *)path {
 }
 
 - (NSString *)storeDictionary:(NSDictionary *)dictionary toPath:(NSString *)path {
-    if ([NSJSONSerialization isValidJSONObject:dictionary]) {
-        NSData *saveData = [NSJSONSerialization dataWithJSONObject:dictionary options:0 error:nil];
-        return [self storeData:saveData toPath:path];
-    } else {
-        [SentryLog logWithMessage:[NSString stringWithFormat:@"Invalid JSON, failed to write file %@", path]
-                         andLevel:kSentryLogLevelError];
-    }
-    return path;
+    NSData *saveData = [SentrySerialization dataWithJSONObject:dictionary options:0 error:nil];
+    return nil != saveData
+            ? [self storeData:saveData toPath:path]
+            : path; // TODO: Should we return null instead? Whoever is using this return value is being tricked.
```

Comment:
is this gonna be addressed or can it be ignored for now?

Reply:
This is the current behavior, I'm again just pointing out the issue.
Same as below